packages/protocol/migrationsConfig.js

```diff
@@ -330,29 +331,53 @@ const NetworkConfigs = {
       frozen: false,
     },
     reserve: {
-      initialBalance: 100000000,
+      initialBalance: 100000000, // CELO
     },
   },
   baklava: {
-    blockchainParameters: {
-      minimumClientVersion: {
-        major: 0,
-        minor: 10,
-        patch: 0,
+    election: {
+      frozen: false,
+    },
+    epochRewards: {
+      targetVotingYieldParameters: {
+        initial: 1.06,
       },
+      frozen: false,
     },
-    election: {
-      minElectableValidators: '25', // About half of the expected genesis set.
+    exchange: {
+      frozen: false,
+    },
+    goldToken: {
+      frozen: false,
     },
     governance: {
       // Set to be able to complete a proposal in about a day, but give everyone a chance to participate.
       dequeueFrequency: 4 * HOUR,
       approvalStageDuration: 4 * HOUR,
       referendumStageDuration: DAY,
       executionStageDuration: WEEK,
+      participationBaseline: 1 / 200, // Very low participation requirement given its a testnet.
+      participationBaselineFloor: 1 / 100,
+      participationBaselineUpdateFactor: 1 / 5,
+      participationBaselineQuorumFactor: 1,
+    },
+    governanceApproverMultiSig: {
+      signatories: [network.from],
```

Comment:
I assume we are ok with anyone who has access to the MNEMONIC to be able to effectively shut down the network?

Reply:
I've gone ahead and created a new account that is loaded into Google Cloud IAM where we can share. Additionally, I went back in git history to add back the signers from RC0, which are still active.